apps/site/src/views/Landing/Landing.tsx

```diff
@@ -1,9 +1,37 @@
-import "./Landing.module.scss";
+"use client";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+
+import ApplyButton from "./components/ApplyButton/ApplyButton";
+import AboutSection from "./components/AboutSection/AboutSection";
+import MentorSection from "./components/MentorSection/MentorSection";
+import SponsorSection from "./components/SponsorSection/SponsorSection";
+import FAQSection from "./components/FAQSection/FAQSection";
+import styles from "./Landing.module.scss";
 
 export default function Landing() {
 	return (
 		<main>
-			<h1>ZotHacks 2023</h1>
+			<Container className="justify-content-center text-center">
+				<Row>
+					<h1>ZotHacks 2023</h1>
+				</Row>
```

Comment:
No need to use a `Row` here for the `h1`. A good rule of thumb to follow is to use them only when you want your content to be in columns.